cpp/ql/lib/semmle/code/cpp/rangeanalysis/new/internal/semantic/analysis/Bound.qll

```diff
@@ -0,0 +1,86 @@
+import cpp
```

Comment:
Is this a copy of some other file?

Reply:
Yep. This is a copy of https://github.com/github/codeql/blob/1df3b017f24a1b8c914c46d8fbcfe161661de243/cpp/ql/lib/experimental/semmle/code/cpp/rangeanalysis/Bound.qll. Maybe we should just delete that file now 🤔

Reply:
f6e774ed2e6cb6ec3366aefe2e2ef85d9b273242 replaces the contents of it an import of the new non-experimental one.